modules/gcp/dc/new_domain_users.ps1.tpl

```diff
@@ -7,6 +7,11 @@
 # Script downloaded from https://activedirectorypro.com/create-bulk-users-active-directory/
 # on 2019.03.22. Modified for Teradici use.
 #
+
+$LOG_FILE = "C:\Teradici\provisioning.log"
+
+Start-Transcript -path $LOG_FILE -append

```

Comment:
can you please make sure Start-Transcript will document which file the code is from?  If it doesn't already write it, please output the name of this file so we know where the logs are from.

Reply:
Confirmed that Start-Transcript will provide a header with a bunch of meta data including which file the code is from:
```**********************
Windows PowerShell transcript start
Start time: 20200420205253
Username: VM-DC\Administrator
RunAs User: VM-DC\Administrator
Configuration Name: 
Machine: VM-DC (Microsoft Windows NT 10.0.17763.0)
Host Application: powershell -file C:/Temp/dc-provisioning.ps1
Process ID: 4200
PSVersion: 5.1.17763.1007
PSEdition: Desktop
PSCompatibleVersions: 1.0, 2.0, 3.0, 4.0, 5.0, 5.1.17763.1007
BuildVersion: 10.0.17763.1007
CLRVersion: 4.0.30319.42000
WSManStackVersion: 3.0
PSRemotingProtocolVersion: 2.3
SerializationVersion: 1.1.0.1
**********************```